boards/arm/adafruit_itsybitsy_m4_express/adafruit_itsybitsy_m4_express_defconfig

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_SAMD51=y
```

Comment:
This SoCs series seems to support MPU peripheral, so you should be enabling CONFIG_ARM_MPU here (unless there are reasons not to do so).

Reply:
Done.  FYI, I have a series of patches that follow on from this that add more peripherals.  There's a few bug fixes to land first.